babel.config.js

```diff
@@ -3,20 +3,56 @@ const config = {
 		[
 			'@babel/env',
 			{
-				corejs: 2,
-				modules: 'commonjs',
 				useBuiltIns: 'entry',
+				corejs: 2,
+				// Exclude transforms that make all code slower, see https://github.com/facebook/create-react-app/pull/5278
+				exclude: [ 'transform-typeof-symbol' ],
 			},
 		],
 		'@babel/react',
 	],
 	plugins: [
-		'@babel/proposal-class-properties',
-		'@babel/transform-runtime',
-		'@babel/transform-member-expression-literals',
-		'@babel/transform-property-literals',
-		'@babel/transform-reserved-words',
+		'@babel/plugin-proposal-class-properties',
+		'@babel/plugin-proposal-export-default-from',
+		'@babel/plugin-proposal-export-namespace-from',
+		'@babel/plugin-syntax-dynamic-import',
+		[
+			'@babel/transform-runtime',
+			{
+				corejs: false, // we polyfill so we don't need core-js
```

Comment:
Does this require us adding `wp-polyfill` as a dependency for Jetpack admin scripts?

Reply:
I think that's the best plan 👍

Reply:
fe755c22e4e098d1370272488b8e127b4acdd451 adds it

Reply:
`corejs: false` means that usages like `Object.assign` or `Promise.resolve` are left as they are. The environment where the code runs must be set up to support these globals and their methods. Including `wp-polyfill` is one way to ensure that.